.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: objective-c
+osx_image: xcode8
+
+before_install:
+- xcrun simctl list # list devices
+
+install:
+- gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+
+script:
+  - xcodebuild -project ISHPullUpSample/ISHPullUpSample.xcodeproj -scheme ISHPullUpSample -destination 'platform=iOS Simulator,name=iPhone 6'  clean build | xcpretty -c || true
```

Comment:
a failing build is not an issue? I'm not sure about the Travis script step, but I think we should set `set -o pipefail` for good measure or `exit ${PIPESTATUS[0]}`